package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@epages/immutable-api-records",
   "description": "ImmutableJS records for the new ePages REST API.",
-  "version": "0.0.77",
+  "version": "0.0.78-rc",
```

```suggestion
  "version": "0.0.78",
```
Remove the `-rc` from the version number for final release after merge.